Gems/ROS2/gem.json

```diff
@@ -29,7 +29,7 @@
         "Atom_Feature_Common",
         "Atom_Component_DebugCamera",
         "CommonFeaturesAtom",
-        "PhysX",
+        "PhysX5",
```

Comment:
We should probably update the `compatible_engines` in this gem.json to reflect the versions of O3DE that will have the PhysX5 gem.

Reply:
Also we should bump the gem version to `3.1.0`